### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by roelbaka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # hyperquaid
-
+```
+Hello
```
Comment (peej) [inline README.md:3]: I believe this is some kind of mistake. Pleasee fix.

Comment (roelbaka) [inline README.md:3]: im sorry the roof is on fire and we decided to let the motherf***er burn

Comment (peej) [inline README.md:3]: image

Comment (roelbaka) [inline README.md:3]: image

**Pull request merged**